src/renderer/mod.rs

```diff
@@ -686,3 +737,54 @@ impl<'a> Renderer<'a> {
         Ok(output)
     }
 }
+
+/// Return a Vec of all substrings contained in '[ ]'s
+/// Ignore quoted strings and integers.
+fn pull_out_square_bracket(s: &str) -> Vec<String> {
+    let mut chars = s.chars();
+    let mut results = vec![];
+    loop {
+        match chars.next() {
+            Some('[') => {
+                let c = chars.next().unwrap();
+                if c != '"' && c != '\'' {
+                    let mut inside_bracket = vec![c];
+                    let mut bracket_count = 1;
+                    loop {
+                        let c = chars.next();
+                        match c {
+                            Some(']') => bracket_count -= 1,
+                            Some('[') => bracket_count += 1,
+                            Some(_) => (),
+                            None => break,
+                        };
+                        if bracket_count == 0 {
+                            // Only store results which aren't numbers
+                            let sub :String = inside_bracket.into_iter().collect();
+                            if sub.parse::<usize>().is_err() {
+                                results.push(sub);
+                            }
+                            break;
+                        }
+                        inside_bracket.push(c.unwrap());
+                    }
+                }
+            },
+            None => break,
+            _ => (),
```

Comment:
I don't believe this is reachable?

Reply:
This is needed for the case of: `Some(_)`